<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryMerger</class>
 <widget class="QDialog" name="BoundaryMerger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge boundary faces</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>351</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="step2_layout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Boundary Merger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="rule_tree">
      <column>
       <property name="text">
        <string>Target name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contain keywords</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>del</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="add_rule">
        <property name="text">
         <string>Add Rule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="merge_subgroup">
        <property name="text">
         <string>Merge Sub-groups</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>720</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>349</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zone list</string>
   </property>
  </widget>
  <widget class="QListWidget" name="zone_list">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>230</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ungrouped</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_left_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_right_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="grouped">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>251</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="new_group_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Group and subgroup</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="ungrouped">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>260</y>
     <width>141</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
